pages/public_cloud/ai_machine_learning/training_tuto_09_train_rock-paper-scissors/guide.en-gb.md

```diff
@@ -0,0 +1,263 @@
+- The ovhai CLI interface installed on your system (more information [here](/pages/public_cloud/ai_machine_learning/cli_10_howto_install_cli))
+- [Docker](https://www.docker.com/get-started) installed and configured to build images.
+- An OCI / Docker image registry. You can use a public registry (such as [Docker Hub](https://hub.docker.com/) for example) or a private registry. Refer to the [Creating a private registry](/pages/public_cloud/containers_orchestration/managed_private_registry/creating-a-private-registry) documentation to create a private registry based on Harbor.
+- Knowledge about building images with [Dockerfile](https://docs.docker.com/engine/reference/builder/)
+
+## Instructions
+
+### Create object storage for data
+
+To train the model you'll need data and a place where to save the trained model.
+You can reuse the previous object storage used in the Notebook tutorial [Train YOLOv8 to play to "rock paper scissors"](/pages/public_cloud/ai_machine_learning/notebook_tuto_15_rock-paper-scissors/) or follow the step [Create Object Storage containers](/pages/platform/ai/notebook_tuto_15_rock-paper-scissors/) of this same tutorial.
+
+### Train your model
+
+> [!primary]
+>
+> All source code is available on the GitHub repository [ai-training-examples]().
+>
+
+To train the model, we will use AI Training. This powerful tool will allow you to automate your pipelines and build fine-tuning phases easily.
+
+AI Training allows you to train models directly from your own Docker images.
+
+First, you need to create a Python script that is in charge of doing the training.
+You can copy and paste the following code in a file named `train-rock-paper-scissors`:
+
+```python
+import ultralytics
+from ultralytics import YOLO
+import shutil
+
+#######################################################################################################################
+## 🎯 The aim of this script is to do transfert learning on YOLOv8 model.                                            ##
+## 💿 The data for train the model are in /workspace/data/rock-paper-scissors/                                       ##
+## 🧠 The train model are stored in /workspace/model/rock-paper-scissors/                                            ##
+#######################################################################################################################
+
+# ✅ Check configuration
+ultralytics.checks()
+
+# 🧠 Load a pretrained YOLO model
+model = YOLO('yolov8n.pt')
+
+# 💪 Train the model with new data ➡️ one GPU / 10 itérations (epochs)
+model.train(data='/workspace/data/rock-paper-scissors/data.yaml', device=0, epochs=50, verbose=True)
+
+# 💾 Save the model
+exportedMetaData = model.export()
+print('Model save to : ' + exportedMetaData)
+
+# ➡️ Copy the model to the object storage
+shutil.copy(exportedMetaData, '/workspace/model/rock-paper-scissors/')
+```
+
+Then, create a `requirements.txt` file to declare the Python dependencies:
+
+```
```

Comment:
Please, update it with the fixed versions

Reply:
done 02f380c163308acddedd17c2313cd4ee23e91c56